packages/react-composites/src/composites/MeetingComposite/adapter/MeetingAdapter.ts

```diff
@@ -93,7 +90,11 @@ export interface MeetingAdapter
       ChatAdapterHandlers,
       'fetchInitialData' | 'sendMessage' | 'sendReadReceipt' | 'sendTypingIndicator' | 'loadPreviousChatMessages'
     >,
-    MeetingAdapterSubscriptions {}
+    MeetingAdapterSubscriptions {
+  joinMeeting(microphoneOn?: boolean): void;
+  leaveMeeting(forEveryone?: boolean): Promise<void>;
+  startMeeting(participants: string[]): void;
```

Comment:
Please document these public methods.

In particular, document that `participants` is an array of ACS user IDs (right?) Are these flattened IDs? Every time we use `string` for user ID in the API, we must explicitly state how to get this string from an actual ID.